{"name": "Проверка ошибки при авторизации с неверным полем", "status": "passed", "steps": [{"name": "Логин курьера в систему", "status": "passed", "parameters": [{"name": "data", "value": "{'login': 'Sasaraf', 'password': '342342'}"}], "start": 1714844796772, "stop": 1714844797521}, {"name": "Проверка статуса и текста", "status": "passed", "parameters": [{"name": "code", "value": "404"}, {"name": "name", "value": "'message'"}, {"name": "text", "value": "'Учетная запись не найдена'"}], "start": 1714844797521, "stop": 1714844797521}], "parameters": [{"name": "login", "value": "'Sasaraf'"}, {"name": "password", "value": "'342342'"}], "start": 1714844796771, "stop": 1714844797523, "uuid": "0e189892-7030-47ea-ad1b-77b0832d677d", "historyId": "642e221e8c02fc8688d1ea7113e0dafd", "testCaseId": "aba13e5b5ebcb4c1ad2d654b19f08135", "fullName": "tests.test_courier_login.TestCourierLogin#test_courier_auth_one_field_wrong", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "Sasaraf"}, {"name": "thread", "value": "19964-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}